### PR TITLE
Add executable test on Windows

### PR DESCRIPTION
**Pull request by ocefpaf**
Make the `xmlwf.exe` available on Windows. (Also re-rendered the recipe.)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,5 +24,6 @@ about:
 
 extra:
     recipe-maintainers:
-        - ocefpaf
         - kwilcox
+        - ocefpaf
+        - pelson
```
Comment (pelson) [inline recipe/meta.yaml]: Sneaky 😄 
👍

Comment (ocefpaf) [inline recipe/meta.yaml]: You wrote that recipe :wink:

Comment (pelson) [inline recipe/meta.yaml]: I better take some responsibility then 😄

**Comment by pelson** [comment]
👍

File: recipe/bld.bat [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+      -D CMAKE_BUILD_TYPE=Release ^
+      .
+if errorlevel 1 exit 1
 
-cmake .. -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release
 nmake install
+if errorlevel 1 exit 1
 
 mkdir %SCRIPTS%
+move %LIBRARY_BIN%\xmlwf.exe %SCRIPTS%\xmlwf.exe || exit 1
```
Comment (ocefpaf) [inline recipe/bld.bat]: This line is the significant change in this PR. (We try to separate re-rendering and recipe changes in different commits.)

**Pull request merged**